bin/cli.js

```diff
@@ -25,18 +28,29 @@ function getCurrentBranch() {
     return execSync('hg id --branch').toString().trim();
 }
 
-function getLatestTag(currentBranch) {
+function getLatestTag() {
+    const currentBranch = getCurrentBranch();
     return execSync(`hg log -r "." -b ${currentBranch} --template "{latesttag}"`).toString().trim();
 }
 
 function getParentRevision() {
     return execSync(`hg log --rev "parents(.)" --template "{rev}" `);
 }
 
+// Removes the "v" at the begining of the current version;
+function getCleanVersion() {
+    return execSync(`hg log -r "." --template "{latesttag}"`).toString().trim().substr(1);
```

Comment:
`execSync(`hg log -r "." --template "{latesttag}"`).toString().trim().replace(/^v/, '')` to replace the first char only if it's a `v`